src/lib/mina_commands/mina_commands.ml

```diff
@@ -356,7 +356,7 @@ let get_status ~flag t =
     in
     match Transition_frontier.catchup_tree frontier with
     | Full full ->
-        Some (Hashtbl.to_alist full.states)
+        Some (List.map (Hashtbl.to_alist full.states) ~f:(fun (state, (n, _hashes)) -> (state, n)))
```

Comment:
This seems to be changing the CLI output. If you plan to merge this, can you include the old vs new output for the affected command, and for completeness can you confirm that the output of the GraphQL endpoint for status is unchanged?

Reply:
This part should convert the `catchup_states` to the original format. I would run a node to test it.